stylesheets/styles.css

```diff
@@ -0,0 +1,112 @@
+.div {
+  width: 800px;
+  height: 600px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+}
+#header {
+  grid-column: 1 / 5;
+  grid-row: 1 / 2;
+  text-align: center;
+}
+
+#navbar {
+  grid-column: 1 / 5;
+  grid-row: 2 / 3;
+  background-color: pink;
+}
+
+nav ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  margin: 0px;
+  padding: 5px 0px 5px 30px;
+}
+
+nav li {
+  display: inline;
+  margin-right: 20px;
+  margin-left: 5px;
+  font-size: 20px;
+}
+nav a {
+  text-decoration: none;
+  color: black;
+  font-family: helvetica, sans-serif;
+}
+
+#projects {
+  grid-column: 1 / 5;
+  grid-row: 3 / 4;
+  font-family: 'Merriweather', serif;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  }
+
+#footer {
+  grid-column: 1 / 5;
+  grid-row: 4 / 5;
+  background-color: pink;
+}
+
+h1 {
+    color: pink;
+    font-family: 'Righteous', cursive;
+    font-size: 50px;
+    -webkit-text-stroke: .1px black;
```

Comment:
Notice how this rule starts with `-webkit`. This is a vendor-specific style tag. We should be really careful about using these in our projects since they're non-standard. It turns out that this is [supported](https://caniuse.com/mdn-css_properties_-webkit-text-stroke) in all the major browsers, but as the note says, we should only use it after careful consideration. It can definitely be frustrating when there may be a style effect that we would like to pull off that has no standard support yet, but it's often preferred to look for work arounds that rely on standard-specified behavior.